hailjwt/hailjwt/hailjwt.py

```diff
@@ -0,0 +1,23 @@
+import jwt
+
+
+class JWTClient:
+    __ALGORITHM = 'HS256'
+
+    def __init__(self, secret_key):
+        self.secret_key = secret_key
+
+    def decode(self, token):
+        if token is None:
+            return None
+        return jwt.decode(
+            token, self.secret_key, algorithms=[JWTClient.__ALGORITHM])
+
+    def encode(self, payload):
+        return jwt.encode(
+            payload, self.secret_key, algorithm=JWTClient.__ALGORITHM)
+
+
+def get_domain(host):
```

Comment:
What is `get_domain` for?

Reply:
I think @akotlar uses it to determine the root domain for the cookies